guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -0,0 +1,235 @@
+
+Under "Development" click the "SQL" button to open the SQL console:
+
+image::oracleautodb/autonomous.database.9.png[]
+
+Copy and paste the following SQL which will create a schema user into the worksheet:
+
+```
+CREATE USER micronautdemo IDENTIFIED BY "XXXXXXXXX";
+GRANT CONNECT, RESOURCE TO micronautdemo;
+GRANT UNLIMITED TABLESPACE TO micronautdemo;
+```
+
+Create a schema user password (must be at least 12 characters and contain a number and an uppercase letter) and replace the text "XXXXXXXXX" with that password.
+
+Click the image:oracleautodb/runscript.png[] ("Run Script") button to execute the SQL:
+
+image::oracleautodb/autonomous.database.10.png[]
+
+== Writing the App
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-app example.micronaut.micronautguide --build=@build@ --lang=@lang@ --jdk=11 --features=oracle,data-jdbc,oracle-cloud-sdk,flyway
+----
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+include::{commondir}/common-default-package.adoc[]
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+=== Dependencies
+
+Oracle Cloud Autonomous Database connection information and credentials are stored in the https://docs.oracle.com/en-us/iaas/Content/Database/Tasks/adbconnecting.htm[Oracle Wallet]. In addition to the dependencies added by the features specified when creating the app, we need a few more for Autonomous Database connectivity and automatic wallet configuration:
+
+dependency:micronaut-oraclecloud-atp[groupId=io.micronaut.oraclecloud]
+
+:exclude-for-build:maven
+[source,groovy]
+----
+implementation platform("com.oracle.database.jdbc:ojdbc-bom:21.1.0.0")
+----
+:exclude-for-build:
+
+:exclude-for-build:gradle
+[source,xml]
+----
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc-bom</artifactId>
+            <version>21.1.0.0</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+----
+:exclude-for-build:
+
+See the https://micronaut-projects.github.io/micronaut-oracle-cloud/latest/guide/[Micronaut Oracle Cloud integration documentation] for more details and options for working with Oracle Cloud in Micronaut apps.
+
+=== Entity class
+
+Create a `Thing` entity class to represent database data:
+
+source:domain/Thing[]
+
+=== Repository class
+
+Create a `ThingRepository` class to read and write `Thing` database data:
+
+source:repository/ThingRepository[]
+
+=== Data populator class
+
+Create a `DataPopulator` class to create some example database entries when the application starts:
+
+source:DataPopulator[]
+
+=== Controller class
+
+Create a `ThingController` class to view persisted data:
+
+source:controller/ThingController[]
+
+=== Configuration
+
+Replace the generated `application.yml` with this:
+
+resource:application.yml[]
```

Comment:
Use an asciidoc tag to not show the complete application.yml but only the relevant parts.

Reply:
it's all relevant, the whole file gets replaced

Reply:
Then probably it's worth explaining why the `micronaut.executors.io.xxxxx` is needed.